src/plugins/azgenerator/CodeModelAzImpl.ts

```diff
@@ -422,12 +491,12 @@ export class CodeModelCliImpl implements CodeModelAz
 
     public get MethodParameter_Name(): string
     {
-        return ToSnakeCase(this.codeModel.operationGroups[this.currentOperationGroupIndex].operations[this.currentOperationIndex].request.parameters[this.currentParameterIndex].language['az'].name);
+        return ToSnakeCase(this.codeModel.operationGroups[this.currentOperationGroupIndex].operations[this.currentMethodIndex].request.parameters[this.currentParameterIndex].language['az'].name);
```

Comment:
this should really come from "python", but as python namer is not integrated yet, i will add an issue to change that later

Reply:
okay